src/app/commands/cmd_new_brush.cpp

```diff
@@ -64,6 +64,10 @@ bool NewBrushCommand::onEnabled(Context* context)
 
 void NewBrushCommand::onExecute(Context* context)
 {
+  ASSERT(current_editor);
+  if (!current_editor)
+    return;
+
```

Comment:
I'll try to find other solution, because a lot of commands can be executed when the Home is selected and a document in other tab.